makefingerprint

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+# makefingerprint generates a video perceptual hash for an input
+SCRIPTDIR=$(dirname $(which "${0}"))
+. "${SCRIPTDIR}/mmfunctions" || { echo "Missing '${SCRIPTDIR}/mmfunctions'. Exiting." ; exit 1 ;};
+SUFFIX="_signature"
+EXTENSION="xml"
+RELATIVEPATH="metadata"
+
+# Remove this section once 'signature' filter is integrated into official FFmpeg release.
+if [ ! -e ~/ffmpeg/ffmpeg ] ; then
```

Comment:
Since it simply skips in this case, I think it's okay. Would prefer that the `~/ffmpeg/ffmpeg` test be more unique since I happen to have a `~/ffmpeg/ffmpeg` but it wouldn't work here. Could it be `~/ffmpegsignature/ffmpeg`?